modular_skyrat/modules/morenarcotics/code/thc.dm

```diff
@@ -1,8 +1,12 @@
-/obj/item/food/grown/cannabis/on_grind()
-	. = ..()
-	if(HAS_TRAIT(src, TRAIT_DRIED))
-		grind_results = list(/datum/reagent/drug/thc/hash = 0.15*src.seed.potency)
-		reagents.clear_reagents() //prevents anything else from coming out
+/obj/item/food/grown/cannabis/grind(datum/reagents/target_holder, mob/user)
+	if(!HAS_TRAIT(src, TRAIT_DRIED))
+		return ..(target_holder, user)
+
+	if(reagents && target_holder)
+		reagents.add_reagent(/datum/reagent/drug/thc/hash, 0.15 * seed.potency)
+		reagents.trans_to(target_holder, reagents.total_volume, transferred_by = user)
+
+	return TRUE
```

Comment:
```suggestion
```

Reply:
On my phone at the moment, but if you're suggesting removing the grind override, see previous comment - hash production is a uniquely stupid chemical reaction meant to allow decimal contribution to making the hash item by individual products.